doc/rst/source/text.rst

```diff
@@ -73,6 +69,10 @@ To add a typeset figure caption for a 3-inch wide illustration, use
     of cities where it is @\_impossible@\_ to get any good Thai food; these are to be avoided.
     EOF
 
+To place a line containing a Latex equation, try::
+
+    echo 3 3 'Use @$\Delta g = 2\pi\rho Gh@$' | gmt text -R0/6/0/6 -JX15c -B -F+f32p+a30 -pdf map
```

Comment:
```suggestion
    echo 3 3 'Use @[\Delta g = 2\pi\rho Gh@[' | gmt text -R0/6/0/6 -JX15c -B -F+f32p+a30 -pdf map
```
There's still a few places with the old syntax, not sure if that's just because the changes are in progress.